mesa/batchrunner.py

```diff
@@ -67,36 +87,42 @@ def __init__(self, model_cls, parameter_values, iterations=1,
 
         self.display_progress = display_progress
 
+    def _process_parameters(self, params):
+        params = copy.deepcopy(params)
+        bad_names = []
+        for name, values in params.items():
+            if (isinstance(values, str) or
+                    not isinstance(values, collections.Sequence)):
```

Comment:
I found an unresolved issue, but with a patch here https://github.com/numpy/numpy/issues/2776#issuecomment-10932295. `Sequence.register(np.ndarray)` has to be added at somewhere.